README.md

```diff
@@ -192,3 +192,17 @@ To do that, go to your **_Account Page_** / **_Pages_**, select the `wildebeest-
 ![unbound](https://imagedelivery.net/NkfPDviynOyTAOI79ar_GQ/45de3429-d01a-4cfc-2ffc-819ac4f51900/public)
 
 After you change your Pages project to Unbound, you need to redeploy it. Go to GitHub Actions in your repo, select the latest successful deploy, and press **Re-run all jobs**.
+
+### WAF false positives
+
```

Comment:
can you please start by explaining what false positive we expect? ie Visible in Firewall event log: ActivityPub endpoint being blocked